packages/adelus/test/vector_random/cxx_main.cpp

```diff
@@ -224,12 +224,11 @@ int main(int argc, char *argv[])
     return 0;
   }
 
-  Kokkos::InitArguments args;
-  args.num_threads = 0;
-  args.num_numa    = 0;
-  args.device_id   = rank%nptile;
+  Kokkos::InitializationSettings args;
+  args.set_num_threads(0);
+  args.set_device_id(rank%nptile);
```

Comment:
@ndellingwood Thanks for updating ```InitArguments``` to ```InitializationSettings```. If it is possible, could you please do similar updates in the ```vector_random_fs``` and ```vector_random_mc``` directories too? Thanks.

Reply:
What does setting num threads to zero even mean?  What is the intent here?

Reply:
@vqd8a I'll add these updates, thanks for feedback!

Reply:
@dalg24 I do not use any thread for this GPU case so I set it to zero. What's your recommendation here?

Reply:
@vqd8a updated with caa9f8078401e2678ce7e60facf8daeae2f3fd43

Reply:
set it to 1

Reply:
@vqd8a I can update to `args.set_num_threads(1);` in this PR if you like

Reply:
@ndellingwood please. Thanks.

Reply:
Updated with e60b1c0807bc945b72d8b3e12d0e2cd0d75b04bf, thanks @vqd8a @dalg24